code/game/objects/items/devices/scanners.dm

```diff
@@ -439,30 +444,29 @@ GAS ANALYZER
 
 		if(ongoing_weather)
 			if((ongoing_weather.stage == MAIN_STAGE) || (ongoing_weather.stage == WIND_DOWN_STAGE))
-				to_chat(user, "<span class='warning'>[src] can't trace anything while the storm is [ongoing_weather.stage == MAIN_STAGE ? "already here!" : "winding down."]</span>")
+				to_chat(user, "<span class='warning'>[src]'s barometer function can't trace anything while the storm is [ongoing_weather.stage == MAIN_STAGE ? "already here!" : "winding down."]</span>")
 				return
 
 			var/time = butchertime((ongoing_weather.next_hit_time - world.time)/10)
-			to_chat(user, "<span class='notice'>The next [ongoing_weather] will hit in [round(time)] seconds.</span>")
+			to_chat(user, "<span class='notice'>The next [ongoing_weather] will hit in [DisplayTimeText(round(time)*10)] seconds.</span>")

```

Comment:
`to_chat(user, "<span class='notice'>The next [ongoing_weather] will hit in [DisplayTimeText(butchertime(ongoing_weather.next_hit_time - world.time))].</span>")`

Reply:
ok the total formula will be `DisplayTimeText(10*butchertime((ongoing_weather.next_hit_time - world.time)/10))`

Reply:
No need to divide and multiply it (which is redundant anyway) with DisplayTimeText since it already handles that math.

Reply:
so accuracy must be divided by 30 then in butchertime